### ISSUE TITLE
Require custom label components to have a specific attribute 

### ISSUE THREAD

**Issue by Larspolo** [vue-a11y/eslint-plugin-vuejs-accessibility#683]
Vue bootstrap makes use of a parent component that requires a `label` attribute before it actually has a label. This is currently not enforcable with the configuration options allowed by form-control-has-label

As an example, see this use case:
https://bootstrap-vue.org/docs/components/form-group#label
```html
    <b-form-group
      label="Enter your name"
      label-for="input-horizontal"
    >
      <b-form-input id="input-horizontal"></b-form-input>
    </b-form-group>
```
